docs/usage.rst

```diff
@@ -73,12 +132,32 @@ Burbach SM, Briney B. Improving antibody language models with native pairing. Pa
 TCR-specific embeddings
 =======================
 
+**Important Note**: Most TCR embedding models listed below are primarily trained on alpha/beta TCRs (TRA/TRB sequences). While AMULETY's unified interface accepts gamma/delta TCRs (TRG/TRD), results may be less reliable for these sequences due to limited training data.
+
 TCR-BERT
 --------
 
-BERT model pre-trained on 88,403 human TCR alpha and beta sequences from VDJdb and PIRD databases. Specialized for T-Cell Receptor analysis.
+BERT model pre-trained on 88,403 human TCR alpha and beta sequences (TRA/TRB) from VDJdb and PIRD databases. Specialized for alpha/beta T-Cell Receptor analysis.
+
+Reference:
+Lu T, Zhang Z, Zhu J, et al. Deep learning-based prediction of the T cell receptor–antigen binding specificity. bioRxiv. 2021. `doi:10.1101/2021.11.18.469186 <https://www.biorxiv.org/content/10.1101/2021.11.18.469186v1>`_
+
+
+
+TCREMP
+------
+
+TCR-specific embedding method trained for T-cell receptor repertoire-based representation learning. Focuses on repertoire-level prediction tasks with specialized TCR sequence understanding.
+
+Reference:
+Zhang H, Zeng T, Zhao Y, et al. TCREMP: T-cell receptor repertoire-based embedding for immunotherapy response prediction. Journal of Molecular Biology. 2025;437: 168712. `doi:10.1016/j.jmb.2025.168712 <https://www.sciencedirect.com/science/article/pii/S0022283625002712>`_
+
+TCRT5
+-----
+
+T5-based model pre-trained on masked span reconstruction using ~14M CDR3 β sequences from TCRdb and ~780k peptide-pseudosequence pairs from IEDB. This model is specialized for TCR beta chains only and supports H chain embedding exclusively.
```

Comment:
We should include models trained on the TCR chain only, not on peptide pairs. Did you include here only the version trained ont the TCR data? We should probably mention this here: e.g. "here we included only the model trained on the TCR sequence only, not the TCR-peptide pairs."